ui/src/lib/access/authorization.ts

```diff
@@ -58,3 +61,7 @@ const isValidUhUuid = (uhUuid: string): boolean => {
     const uhUuidPattern = new RegExp(/^[0-9]{8}$/);
     return uhUuidPattern.test(uhUuid);
 };
+
+const isDepartmental = (user: User): boolean => {
+    return user.uid == user.uhUuid || user.uid == null || user.uhUuid == null;
```

Comment:
The definition for a department account should be `user.uid == user.uhUuid || user.uhUuid == ""`, based on our Angular.js code. To play it safe, I think a isEmpty() function should be added to /lib/utils.ts and it should replace user.uhUuid == null with isEmpty(user.uhUuid).
```
const isEmpty = () => {
  return value === null || value === undefined || value === "";
}
```

Reply:
Good point, I didn't think of the other cases like an empty string or an undefined value.